ssl.go

```diff
@@ -64,13 +68,13 @@ func ssl(o values) func(net.Conn) net.Conn {
 // in the user's home directory. The configured files must exist and have
 // the correct permissions.
 func sslClientCertificates(tlsConf *tls.Config, o values) {
-	sslkey := o.Get("sslkey")
-	sslcert := o.Get("sslcert")
+	sslkey, sslkeyOK := o["sslkey"]
+	sslcert, sslcertOK := o["sslcert"]
 
 	var cinfo, kinfo os.FileInfo
 	var err error
 
-	if sslcert != "" && sslkey != "" {
+	if sslkeyOK && sslcertOK {
```

Comment:
The old code is also incompatible with that `libpq` logic in that `libpq` allows either the certificate or the key to be blank, while this only does the right thing if both are blank.

I'm inclined to "break" compatibility in this way until someone reports that they need it, at which time we should add tests and implement the actual `libpq` behaviour.